### DISCUSSION TITLE
Headtracking verschoben

### DISCUSSION THREAD

**Discussion by racer001** [Mogli12/KeyboardSteer#736] [Previous version]
Hallo,

Ich nutze den Mod und Tobii eyex schon länger, hier vorab ein fettes DANKE für diese Mod! 

Mir ist aufgefallen dass bei einigen Fahrzeugen (egal ob Mod oder auch manche Vanilla) die Kamera beim umsehen auf die Motorhaube verschoben wird.

Es scheint sich mit dem Eye(Head-)tracking zu beißen, wenn ich Headtracking in den Einstellungen deaktiviere funktioniert der Mod mit den Fahrzeugen wo der Fehler davor aufgetreten ist ohne Probleme. Ist das Problem bekannt und gibt es dafür einen fix? Mir gefällt die umdrehfunktion ganz gut weil man mit dem Eyetracking nicht so schnell ist wie mit Headtracking.

Grüße
Racer001

**Comment by templer013** [comment]
Hi, 
danke für diese super Mod! (und die vielen vorher!)

LS19, nur VCA im Mod Ordner, Vanilla Map und Fahrzeug. 
Genau das gleiche Problem hier. Wenn Head Tracking in den Einstellungen aktiviert, dann sitzt man bei aktivierter Rückfahrkamera auf der Motorhaube. 

Fendt Favorit 515C: Position auf Motorhaube
Claase Arion 660: Position auf Motorhaube
Rostselmash 330: Position im Fruchttank (also nach hinten verschoben anstatt nach vorne, wie bei den anderen beiden)

VG
templer013

**Comment by Mogli12** [comment]
Du kannst doch die Kamerabewegung durch VCA komplett abstellen. Das wird doch eh vom Headtracking übernommen. Ich verstehe nicht ganz, warum man beides gleichzeitig an hat.

**Comment by templer013** [comment]
Hi, 
wenn man Rückwärts fährt, dann ist das mit Head Tracking äußert ungeschickt, weil man dabei ständig quasi den Kopf zur Seite gedreht gedreht hält... Ich war begeistert, als ich feststellte, dass über Mods die Kamera in der Kabine noch beeinflusst werden kann. Giants deaktiviert ja jegliche Kamera Aktivität jenseits des Head Tracking Inputs.
